src/main/resources/config/application.yml

```diff
@@ -10,7 +10,6 @@ security:
       tokenInfoUri: ${TOKENINFO_URL}
 
 management:
-  port: 7979
```

Comment:
@vroldanbet yes, please revert the management port and overwrite in test classpath if necessary